### ISSUE TITLE
Update the Windows Icon from that orange box to Jaime

### ISSUE THREAD

**Issue by MeFisto94** [jMonkeyEngine/sdk#19]
This Issue is quite simple but since I'm not under Windows right now, I'll open this Issue for anyone to fix/as a reminder for me:
https://forums.netbeans.org/topic48135.html

In short: Use ResHack to change the Icon of `netbeans/harness/launchers/app.exe`
Note: That app is downloaded automatically so we need some "override" and thus manually update the icon each new netbeans version (shouldn't be a problem, though it leads to bad errors when not done).

Maybe using https://github.com/erocarrera/pefile for editing? However I feel bad in adding python as a dependency just for that. Maybe there's a nice java solution?

https://github.com/mirror/launch4j/blob/master/src/net/sf/launch4j/RcBuilder.java This is launch4j's RCBuilder however those .rc's have to be compiled (e.g. by `wrc`).

**Comment by MeFisto94** [comment]
We need Jaime in this Dimensions:
```
48 × 48 (256 colors) - Ordinal name: 1
32 × 32 (256 colors) - Ordinal name: 2
16 × 16 (256 colors) - Ordinal name: 3
48 × 48 (16.8mil colors) - Ordinal name: 4
32 × 32 (16.8mil colors) - Ordinal name: 5
```
I can do the downsampling however where to place them then?

**Comment by MeFisto94** [comment]
Completely forgot to adress this issue in the commit...
Solved in dfe6cbb though an automated solution would be nice to have, so feel free to re-open this issue when you've found a way to _easily_!! change PE resources from Java Code (without many bloaty libs, installing wine/python/ what ever).
